Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml

```diff
@@ -245,12 +285,14 @@ tasks:
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
-  "7":
-    id: "7"
-    taskid: 62dda88b-b3a5-44c2-8fba-30516903d664
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 9228e6e2-2e0f-493c-8963-397b35f924bc
     type: regular
     task:
-      id: 62dda88b-b3a5-44c2-8fba-30516903d664
+      id: 9228e6e2-2e0f-493c-8963-397b35f924bc
       version: -1
       name: Set Empty List
       description: commands.local.cmd.list.set
```

Comment:
This is the OOTB description of this automation?

Reply:
I think that it's a server script and this is why the description looks like that